packages/nimble-components/src/wafer-map/data-types/WaferMapDie.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * WaferMapDie
+ */
+export class WaferMapDie {
+    public data!: string | number;
+    public x!: number;
+    public y!: number;
+}
```

Comment:
I expect users will create the data by doing the following:

```ts
const die: WaferMapDie = {
    data: '12',
    x: 3,
    y: 4
};
```

Instead of by doing:
```ts
const die = new WaferMapDie();
die.data = '12;
die.x = 3;
die.y = 4;
```

If that is the case then we should instead define this as an interface and move it to `types.ts`

```ts
export interface WaferMapDie {
    data: string | number;
    x: number;
    y: number;
}
```

Reply:
I updated and I did the same also for the "WaferMapColorsScale" class